Client/Frontend/Settings/SyncContentSettingsViewController.swift

```diff
@@ -156,6 +156,9 @@ class SyncContentSettingsViewController: SettingsTableViewController, FeatureFla
             if engineName == .creditcards {
                 self.creditCardSyncEnabledTelemetry(status: enabled)
             }
+            else if engineName == .passwords {
```

Comment:
nit, easier to read that way I think
```suggestion
            } else if engineName == .passwords {
```